content/en/docs/concepts/cluster-administration/api-priority-and-fairness.md

```diff
@@ -0,0 +1,138 @@
+This includes not only kube-apiservers but aggregated API servers too.
+
+Prioritization and fairness applies only to requests that are not
+_long-running_.  The nature of the request, not the size of its
+reply, determines whether a request is long-running.  The requests
+considered to be long running are
+[WATCHES](https://kubernetes.io/docs/reference/using-api/api-concepts/#efficient-detection-of-changes),
+[EXECs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#exec),
+and
+[fetching of logs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs)
+(with or without "following").
+
+Each API server independently applies prioritization and fairness,
+based on API objects that configure this feature.  This involves two
+concepts: priority level and flow schema.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Enabling API priority and fairness
+
+API priority and fairness is an alpha feature, so it is disabled by
+default.  To enable API priority and fairness you need to do two
+things: turn on the RequestManagement feature gate (e.g., by including
+the `--feature-gates RequestManagement=true` flag when starting your
+API servers) and enable the flowcontrol.apiserver.k8s.io/v1alpha1 API
+version (e.g., by including the `--runtime-config
+flowcontrol.apiserver.k8s.io/v1alpha1=true` flag when starting your
+`kube-apiserver`s).
+
+Enabling the feature gate replaces the "max in flight" filter, which
+enforces simple aggregate concurrency limits, with a filter that
+implements API priority and fairness --- which can be seen as a more
+general version of the "max in flight" filter.
+
+## Priority level
+
+Each API server has an aggregate concurrency limit and divides
+it&mdash;without examining the request load&mdash;among several
+priority levels.  The priority levels operate independently of each
+other.  That is, the API server classifies a given request to exactly
+one priority level and, for a given priority level, handles at most
+the corresponding number of requests at any given time.  One special
+priority level&mdash;used for meta-requests and other top-priority
+traffic&mdash;imposes no limit.  This priority level and the requests
+mapped to it are described as _exempt_, because of the lack of
+concurrency limitation.
+
+For a non-exempt request, if it is not allowed to be handled
+immediately then the API server will either reject the request or hold
+it in a queue for a while.  Each non-exempt priority level has a
+number of FIFO queues.  Whenever that priority level's number of
+requests currently being handled drops below the concurrency limit,
+the API server draws requests to begin handling from these queues in a
+fair way.
+
+## Flow schema
+
+To determine how to control a request, an API server first classifies
+that request to a _flow_ according to a configured collection of _flow
+schemas_.  The flow schemas are considered in order, according to each
```

Comment:
>  The flow schemas are considered in order, according to each flow schema's _matching precedence_ (which is a number).

I recommend writing in the active voice, for example:
```markdown
Each FlowSchema is a set of rules along with a _matching precendence_. The
API server evaluates the FlowSchemas in order, sorted based on based on their
matching precedence (an integer).
A flow schema also includes the name of the priority level to use for
requests that match.
```
(is it an integer? I guessed. The reader would want to know.)